### ISSUE TITLE
Expose multiple predefined templates

### ISSUE THREAD

**Issue by narakhan** [astahmer/openapi-zod-client#130]
There are several common usecases which the library could add first-party template support for, one example being the already written [schemas-only.hbs](https://github.com/astahmer/openapi-zod-client/blob/3933722e0c6d7a7b7af50dd63d31aa5f91f666a1/lib/src/templates/schemas-only.hbs) template. This functionality would slot into the existing CLI/API simply by modifying `templatePath` to be of the type `undefined | "schemas-only" | ... | string` and extending the existing TS-Pattern `match` statement to check for the literal strings of the predefined templates.

**Comment by astahmer** [comment]
good idea ! feel free to open a PR for that (with a [changeset](https://github.com/changesets/changesets/blob/main/docs/intro-to-using-changesets.md))
